pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -531,6 +531,44 @@ type PostgresVolumesSpec struct {
 	// ---
 	// +optional
 	Temp *VolumeClaimSpec `json:"temp,omitempty"`
+
+	// Additional pre-existing volumes to add to the pod.
+	// ---
+	// +optional
+	// +listType=map
+	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=10
+	Additional []*AdditionalVolume `json:"additional,omitempty"`
```

Comment:
🤔 Is pointer helpful here? Do we want to be doing `nil` checks?


```suggestion
	Additional []AdditionalVolume `json:"additional,omitempty"`
```